src/Asset/FrontendAsset.php

```diff
@@ -8,36 +8,54 @@
 
 namespace HeimrichHannot\FlatpickrBundle\Asset;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use HeimrichHannot\UtilsBundle\Container\ContainerUtil;
 
 class FrontendAsset
 {
     /**
-     * @var ContainerInterface
+     * @var \HeimrichHannot\EncoreBundle\Asset\FrontendAsset
      */
-    private $container;
+    protected $encoreFrontendAsset;
+
+    /**
+     * @var ContainerUtil
+     */
+    protected ContainerUtil $containerUtil;
 
     /**
      * FrontendAsset constructor.
      */
-    public function __construct(ContainerInterface $container)
+    public function __construct(ContainerUtil $containerUtil)
     {
-        $this->container = $container;
+        $this->containerUtil = $containerUtil;
+    }
+
+    public function setEncoreFrontendAsset(\HeimrichHannot\EncoreBundle\Asset\FrontendAsset $encoreFrontendAsset): void
+    {
+        $this->encoreFrontendAsset = $encoreFrontendAsset;
     }
 
     public function addFrontendAssets()
     {
-        if (!$this->container->get('huh.utils.container')->isFrontend()) {
+        if (!$this->containerUtil->isFrontend()) {
             return;
         }
 
-        if ($this->container->has('huh.encore.asset.frontend')) {
-            $this->container->get('huh.encore.asset.frontend')->addActiveEntrypoint('contao-flatpickr-bundle');
-            $this->container->get('huh.encore.asset.frontend')->addActiveEntrypoint('contao-flatpickr-bundle-theme');
+        if ($this->encoreFrontendAsset) {
+
+            if(!$this->encoreFrontendAsset->isActiveEntrypoint('contao-flatpickr-bundle')) {
+                $this->encoreFrontendAsset->addActiveEntrypoint('contao-flatpickr-bundle');
+            }
+
+            if(!$this->encoreFrontendAsset->isActiveEntrypoint('contao-flatpickr-bundle-theme')) {
+                $this->encoreFrontendAsset->addActiveEntrypoint('contao-flatpickr-bundle-theme');
+            }
+
+        } else {
```

Comment:
Der Code muss nicht in ein else, da Encore die entries automatisch raus schmeißt